### PR TITLE
Log in

### LINKED ISSUE #7: Log in Page
- [x] Create Login Page

### PR DESCRIPTION
**Pull request by akbarjorayev**
- [x] Create Login Page

### CODE REVIEW

**Review by akbarjorayev — COMMENTED** [review]
**Finish** the `Login` page

**Review by akbarjorayev — COMMENTED** [review]
**LGTM** @nazarovctrl

**Comment by akbarjorayev** [comment]
@nazarovctrl merge?

File: src/js/utils/checker.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export function isValidUsername(username) {
```
Comment (nazarovctrl) [inline src/js/utils/checker.js]: keremas alohia fayl qilish

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

File: src/js/utils/checker.js (hunk 1 above)
Comment (akbarjorayev) [reply src/js/utils/checker.js]: Kelajakda yana narsalar qoshilishi mumkin...

Comment (nazarovctrl) [reply src/js/utils/checker.js]: Ishlatishmidi barbir bu boshqa joyda

Comment (akbarjorayev) [reply src/js/utils/checker.js:1]: ok

**Review by akbarjorayev — COMMENTED** [review]
**LGTM**

File: src/pages/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    console.log(loginData);
+  }
+
+  function checkValidation() {
+    const passwordVal = passwordInput.current.value;
+    const [username, companyName] = usernameInput.current.value.split('@');
+
+    setValidForm(
+      isValidUsername(username) &&
+        isValidUsername(companyName) &&
```
Comment (nazarovctrl) [inline src/pages/Login/Login.jsx]: Nega  isvalidusername ga companyname bervosan?

Keyin vapsheta   
azimjon@bandla
San shuni ozini Yani loggini ni tekshir boldi
azimjon ni alohida  bandla ni alohida tekshirma

Example
azimjon@bandla -> valid
azimjonbandla -> not valid 
azimjonbandla@bandla -> valid

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

File: src/pages/Login/Login.jsx (hunk 2 above)
Comment (akbarjorayev) [reply src/pages/Login/Login.jsx]: Example code

Comment (akbarjorayev) [reply src/pages/Login/Login.jsx]: > Nega isvalidusername ga companyname bervosan?

Company name tekshirish keremasmi?

Comment (nazarovctrl) [reply src/pages/Login/Login.jsx]: Bitta qb tekshir dvomau , alohida qimasdan

**Review by nazarovctrl — APPROVED** [review]

**Pull request merged**